README-zh.md

```diff
@@ -1,27 +1,27 @@
-﻿![Coil](logo.svg)
+![Coil](logo.svg)
 
-Coil是一个Android图片加载库，通过Kotlin协程的方式加载图片。特点：
+Coil 是一个 Android 图片加载库，通过 Kotlin 协程的方式加载图片。特点：
```

Comment:
Are the spaces needed for correct grammer (I don't speak the language)?

Reply:
Yeah, I' m chinese, should leave blanks between chinese chars and english chars in formal format

Reply:
Not only added blanks, but also modified some translation parts